duckduckgo/Android
# Remove the DummyInternalFeature

Task/Issue URL: https://app.asana.com/0/414730916066338/1200887280779475/f
Tech Design URL: 
CC: 

**Description**:
We added the dummy internal feature to showcase how to add internal feature entry points when we didn't have any.
Now we've got already some internal feature so the dummy feature is no longer useful.


**Steps to test this PR**:
1. install the internal release/debug build from this branch
1. go to Settings -> scroll down
1. verify dummy feature is not in the list of internal features


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
